src/cmdline.rs

```diff
@@ -783,7 +813,9 @@ pub struct IsoResetConfig {
 }
 
 #[derive(Debug, StructOpt)]
-pub struct OsmetPackConfig {
+// default usage line lists all mandatory options and so exceeds 80 characters
+#[structopt(usage = "coreos-installer pack osmet [OPTIONS]")]
```

Comment:
Seems like overkill to enforce the 80 character rule even for these commands. Though meh, I guess it's good to be consistent.

Reply:
Yeah, part of the point of improving discoverability is having readable help, so I think it's worth the hassle.